<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xLabel">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="xLineEdit">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yLabel">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="yLineEdit">
        <property name="text">
         <string>-1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="zLineEdit">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="magLabel">
        <property name="text">
         <string>Mag</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="magLineEdit">
        <property name="text">
         <string>1.0e2</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="useSubCheckBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="useSubLabel">
        <property name="text">
         <string>useSub</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="nRLineEdit">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="nRLabel">
        <property name="text">
         <string>NR</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="regLineEdit">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="regLabel">
        <property name="text">
         <string>reg</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="yLabel_2">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="yLineEdit_2">
        <property name="text">
         <string>1e6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rhoLabel">
        <property name="text">
         <string>rho</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rhoLineEdit">
        <property name="text">
         <string>0.33</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Optimize</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize Internal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize Surface</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>menuBar</sender>
   <signal>triggered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>pressAction(QAction*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>applyForce()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>doOptimize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDrawInternal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setDrawSurface()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pressAction(QAction*)</slot>
  <slot>applyForce()</slot>
  <slot>doOptimize()</slot>
  <slot>setDrawSurface()</slot>
  <slot>setDrawInternal()</slot>
 </slots>
</ui>
